utilities.lib.nut

```diff
@@ -189,8 +207,7 @@ utilities._extractUUID <- function(rs) {
 // **********         Public           **********
 utilities.debugI2C <- function(i2c) {
     if (imp.environment() == ENVIRONMENT_AGENT) {
-        server.error("utilities.debugI2C() can only be run on a device");
-        return;
+        throw "utilities.debugI2C() can only be run on a device";
     }
 
     for (local i = 2 ; i < 256 ; i+=2) {
```

Comment:
This scan method doesn't work on the imp005 (at least it didn't for me when I was writing tests for AutoDesk accelerator)- may want to note this in the docs.

Reply:
Will check this